src/blocks/container/backgroundCss.js

```diff
@@ -159,7 +159,7 @@ const backgroundCss = ( attributes, deviceType ) => {
 		);
 
 	for ( const cssProp in bgCss ) {
-		createCamelCase[ snakeToCamel( cssProp ) ] = bgCss[cssProp].replace( ';', '' );
+		createCamelCase[ snakeToCamel( cssProp ) ] = bgCss[cssProp]?.replace( ';', '' );
```

Comment:
```suggestion
if( ! bgCss[cssProp] ){
 continue;
}
		createCamelCase[ snakeToCamel( cssProp ) ] = bgCss[cssProp].replace( ';', '' );
```
Please verify if css value is null or empty then also css markup will generate with empty 
Ex- `background-image : ''`